gdx-fireapp-android/src/test/java/pl/mk5/gdx/fireapp/android/TestUtils.java

```diff
@@ -0,0 +1,39 @@
+package pl.mk5.gdx.fireapp.android;
+
+import android.os.Bundle;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class TestUtils {
```

Comment:
two comments,

1. I think we can make it package-private
2 We should add private constructor here. There is only one static method

Reply:
Thanks for the code review. I applied both suggestions 👍 Sorry for the slow fix, just launched the above-mentioned game so it's been a little hectic!